src/controllers/comments/deleteCommentById.js

```diff
@@ -0,0 +1,18 @@
+const { deleteCommentByIdQuery } = require('../../database/queries');
+
+const deleteCommentById = (req, res, next) => {
+  const { id } = req.params;
+  deleteCommentByIdQuery(id)
+    .then(() => {
+      res.status(200).json({
+        error: false,
+        data: {
+          message: 'Comment Deleted successfully',
+        },
+      });
+    })
```

Comment:
Consider implementing proper authorization and access control checks to ensure that only authorized users can delete comments. This can involve checking if the user making the request has the necessary permissions to delete the comment.

To prevent unauthorized deletion of comments, implement user-specific authorization checks. By verifying the user's credentials and ensuring they have the necessary permissions, only authorized users can delete comments they own. This ensures data integrity and protects against unauthorized modifications.

What I mean:
You should check if the user's request to delete the comment is the same user he/she adds this comment, and if the user is not the same you should send an error message.